src/peripheral/lift_server_node.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file  action_server.cpp
+ * @brief This file contains the implementation of a simple action and service
+ *        server for lift action. It communicates with an Arduino via a serial
+ *        port to transmit messages (e.g., goals, feedback request) and receive
+ *        feedback.
+ *
+ * @date  09-01-2024
+ *
+ * @copyright Copyright (c) 2024 Weston Robot Pte. Ltd.
+ */
+
+#include "wrp_ros/peripheral/lift_server_node.hpp"
+
+namespace westonrobot {
+ActionServer::ActionServer()
+    : liftController_(),
+      Server_(
+          nh_, "lift_action",
+          [this](const wrp_ros::LiftControlGoalConstPtr& goal) {
+            this->ExecuteCB(goal);
+          },
+          false) {
+  const char* device_path = FindSerialPort();
+  liftController_.Connect(device_path, baud_rate);
```

Comment:
You specified in readme that device_path and baud_rate are ROS parameters but are not actually exposing these as parameters.